readme.md

```diff
@@ -104,8 +115,8 @@ q.start(function (err) {
 ## Install
 `npm install queue`
 
-_Note_: You may need to install the [`events`](https://github.com/Gozala/events) dependency if 
-your environment does not have it by default (eg. browser, react-native). 
+_Note_: You may need to install the [`events`](https://github.com/Gozala/events) dependency if
+your environment does not have it by default (eg. browser, react-native).
```

Comment:
just picking nits here, but generally to keep the history clean i prefer if stuff like this gets its own commit